<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 218, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="red_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="green_slider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blue_slider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Red">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Green">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Blue">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="color">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="red_num">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="green_num">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="blue_num">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
